tools/syz-declextract/run.go

```diff
@@ -501,6 +761,14 @@ func (ctx *context) appendNodes(nodes []ast.Node, file string) {
 				} else {
 					ctx.mergeInterface(iface)
 				}
+			case strings.HasPrefix(node.Text, "JSON:"):
```

Comment:
This look really hacky.

Is it possible to make the C++ part take an argument with a file name to write the JSON to? Or make it output some structured format, some part of which will contain syzlang and the other part JSON?

Reply:
I will need to refactor it significantly to make it non hacky. At this point adding most things to the clang tool or run.go is hacky. The "INTERFACE:" thing used the same hack. We need to switch everything to json.

Reply:
Ack. Let's leave some TODO comment for now.

Reply:
Superseded by non-hacky version in #5583